File: Arrays/Task2.java
import java.util.Arrays;
public class Task2 {
	public static int[] reversealist(int[] number) {
	
	int arrayBeginning =0;
	int arrayEnd = number.length -1;

	while (arrayBeginning  < arrayEnd){
	int swap = number[arrayBeginning];
	number[arrayBeginning] = number[arrayEnd];
	number[arrayEnd] = swap;
	arrayBeginning++;
	arrayEnd--;
	}
	
	//Collections.reverse(number);
	return number;
}

	public static void main(String[] args){
	int[] number = {1, 2, 3, 4, 5,};
	//Collections.reverse(number) = reversealist(number);
	int[] reversealist = reversealist(number);
	System.out.print(Arrays.toString(number));
	}
}

File: pythonclass/Arrays/task2.py
def reverse_a_list(number):
        arraybeginning =0
        arrayend = len(number) - 1

        while arraybeginning < arrayend:
            number[arraybeginning], number[arrayend] = number[arrayend], number[arraybeginning]
            number[arraybeginning] += 1
            number[arrayend] -= 1
        return number
print(reverse_a_list([1, 2, 3, 4, 5]))
            
